src/styles/ForecastDetails.css

```diff
@@ -0,0 +1,21 @@
+.forecast-details {
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: aliceblue;
+  align-self: center;
+  width: fit-content;
+  border-radius: 5%;
+  border-color: #850707;
+  border-style: solid;
+  border-width: 1px;
+}
+
+.forecast-details > div {
+  padding: 5px;
+}
+
+.forecast-details_wind > img {
+  height: 1em;
```

Comment:
A note about units here, its best to use rem for font sizes (youre using this image like a font here so that would apply in this case), em for things like margin and padding, and px for small things like border widths. This will be covered more in the styling lectures later on so dont worry about it too much right now, for reasons why have a look at [this article](https://dev.to/theodorusclarence/back-to-basic-should-we-use-rem-em-or-pixel-1hd0)